test/PackageUtilities.js

```diff
@@ -180,55 +231,4 @@ describe("PackageUtilities", () => {
       });
     });
   });
-
-  describe(".filterPackages()", () => {
```

Comment:
Somehow we got two blocks of `filterPackages` tests. I condensed them, using the public API instead of the "private" underscore-prefixed method.